src/shared/utils/__tests__/utils.test.ts

```diff
@@ -387,139 +230,91 @@ describe("Util Functions", () => {
-          privacy: {
-            obfuscated: [],
+    it("should return false for a v2 DID wrapped document without document store", () => {
+      expect(utils.isDocumentRevokable(v2WrappedDidDocument)).toStrictEqual(false);
+    });
+    it("should return true for a v3 DID wrapped document when revocation store is 'REVOCATION_STORE'", () => {
+      const document: any = {
+        ...v3WrappedDidDocument,
+        openAttestationMetadata: {
+          ...v3WrappedDidDocument.openAttestationMetadata,
+          proof: {
+            ...v3WrappedDidDocument.openAttestationMetadata.proof,
+            revocation: {
+              type: v3.RevocationType.RevocationStore,
+            },
           },
         },
       };
-      expect(utils.isTransferableAsset(document)).toStrictEqual(false);
+
+      expect(utils.isDocumentRevokable(document)).toStrictEqual(true);
+    });
+    it("should return false for a v3 DID wrapped document when revocation store is 'NONE'", () => {
+      expect(utils.isDocumentRevokable(v3WrappedDidDocument)).toStrictEqual(false);
     });
   });
```

Comment:
missing 1 more test case which u mentioned about having a document store but without proof value

Reply:
ok will add that in